arbos/programs/programs.go

```diff
@@ -250,7 +266,10 @@ func (p Programs) CallProgram(
 	if runmode == core.MessageCommitMode {
 		arbos_tag = statedb.Database().WasmCacheTag()
 	}
+
+	metrics.GetOrRegisterCounter(fmt.Sprintf("arb/arbos/stylus/program_calls/%s", runModeToString(runmode)), nil).Inc(1)
 	ret, err := callProgram(address, moduleHash, localAsm, scope, interpreter, tracingInfo, calldata, evmData, goParams, model, arbos_tag)
+	gasUsed := callCost
```

Comment:
callCost is just for the initial cost charged.
There is more charging going on inside callProgram.
I think the best/simplest way to see how much gas was actually charged is looking at startingGas - contract.Gas (just before returning from the function)

Reply:
Done.